modules/tidb/build.gradle

```diff
@@ -0,0 +1,19 @@
+description = "Testcontainers :: JDBC :: TiDB"
+
+dependencies {
+    annotationProcessor 'com.google.auto.service:auto-service:1.0.1'
+    compileOnly 'com.google.auto.service:auto-service:1.0.1'
+
+    api project(':jdbc')
+
+    compileOnly project(':r2dbc')
+    compileOnly 'dev.miku:r2dbc-mysql:0.8.2.RELEASE'
```

Comment:
there is no tests for r2dbc nor implementation for it. Can you add them or remove those dependencies, please?

Reply:
Deleted